mimikatz/modules/token/list.md

```diff
@@ -1,7 +1,10 @@
 # list
 
-`token::list` lists all tokens on the system. It has the following command line arguments:
+`token::list` lists all tokens on the system.
 
+It has the following command line arguments:
+
+* `/token`: The token to list
```

Comment:
Can you give an example in this page? I only see examples of `/user:`, `/admin`, `id:`, `/domainadmin`, `/localservice`, `/enterpriseadmin`

Reply:
This was a mistake. It's actually called `/id` and not `/token`. Good catch. Fixed in 70be116.